### PR TITLE
added overwriteFormulaCells parameter

### PR DESCRIPTION
**Pull request by spoltier**
added XLC-java from https://github.com/miraisolutions/xlconnect-java/actions/runs/3046949308

### CODE REVIEW

**Comment by spoltier** [comment]
@Chargothrond this branch has the jar with the `overwriteFormulaCells` parameter added, it should be possible to add this parameter to the existing R functions (had to also add it to `append...` methods) to fix the CI, in addition to probably creating a test from Martin's example on #173

File: inst/unitTests/runit.workbook.appendNamedRegion.R [hunk 1]
```diff
@@ -28,6 +28,13 @@
 
 test.workbook.appendNamedRegion <- function() {
 	
+  test_overwrite_formula <- function(wb, expected, coords, overwrite = TRUE) {
```
Comment (martinstuder) [inline inst/unitTests/runit.workbook.appendNamedRegion.R:31]: Indentation looks strange here

File: inst/unitTests/runit.workbook.writeNamedRegion.R [hunk 2]
```diff
@@ -1,75 +1,100 @@
...
+	createName(wb.xls, "nope", "NonExistingSheet!A1")
+	checkException(writeNamedRegion(wb.xls, mtcars, "nope"))
+	
+	# Check that attempting to write to a name which referes to a non-existing sheet
+	# causes an exception (*.xlsx)
+	createName(wb.xlsx, "nope", "NonExistingSheet!A1")
+	checkException(writeNamedRegion(wb.xlsx, mtcars, "nope"))
+  
+	# Check that writing an empty data.frame does not cause an error (*.xls)
+  createSheet(wb.xls, "empty")
```
Comment (martinstuder) [inline inst/unitTests/runit.workbook.writeNamedRegion.R:62]: Indentation

**Comment by martinstuder** [comment]
:100:  @spoltier and @Chargothrond !

**Comment by spoltier** [comment]
@martinstuder @Chargothrond there was a patch release of POI not long ago, I've upgraded to that on both PRs

**Review by martinstuder — APPROVED** [review]

**Pull request merged**